### PR TITLE
Add pagination to the change tables

### PR DESCRIPTION
**Pull request by mhuin**

### CODE REVIEW

File: web/src/components/Change.res [hunk 1]
```diff
@@ -264,16 +264,70 @@ module RowItem = {
...
+    let (perPage, setPerPage) = React.useState(_ => 25)
+    let (changeArray, _) = React.useState(_ => Belt.List.toArray(changes))
+    let onSetPage = (_, pageNumber: int, _, _, _) => {
+      setPage(_ => pageNumber)
+    }
+    let onPerPageSelect = (_, perPage: int, _, _, _) => {
+      setPerPage(_ => perPage)
+    }
+    <>
+      <Pagination
```
Comment (TristanCacqueray) [inline web/src/components/Change.res:317]: On the board page there can be multiple tables, and hopefully most of them should not be too crowded, so perhaps we should hide the pagination when the list is less than 50 elements? You could use the `React.null` component in that case.

File: web/src/components/Change.res [hunk 2]
```diff
@@ -264,16 +264,70 @@ module RowItem = {
...
+    ~perPage: int=?,
+    ~variant: @string
+    [
+      | @as("top") #Top
+      | @as("bottom") #Bottom
+      | @as("PaginationVariant") #PaginationVariant
+    ]=?,
+    ~widgetId: string=?,
+  ) => React.element = "Pagination"
+}
```
Comment (TristanCacqueray) [inline web/src/components/Change.res:302]: The binding is now part of re-patternfly-3.0.2, could you please update the package.json (and run npm install after) instead?
```suggestion
```

**Review by TristanCacqueray — CHANGES_REQUESTED** [review]
Thank you for the PR, this is looking great.

Monocle can display the changes with another component for the `Browse Changes` page, so we need to add the pagination there too. Unfortunately the component is not standalone and it can't be displayed in the designer view, it's: https://github.com/change-metrics/monocle/blob/1d0100250cfcb09f64688e16ce4f405cde2e320d/web/src/components/NChangeView.res#L155-L161

So could you please:

- Extract the jsx into a new `NChangeView.ChangeList` component
- Add the ChangeList to the designer
- Add pagination to the ChangeList

File: web/src/components/Change.res (hunk 1 above)
Comment (mhuin) [reply web/src/components/Change.res]: Done

File: web/src/components/Change.res (hunk 2 above)
Comment (mhuin) [reply web/src/components/Change.res]: Done

**Review by TristanCacqueray — APPROVED** [review]
Thanks, in 110f377 I took the liberty to introduce a new usePagination hook to consolidate the logic. Let me know what you think about it!

**Pull request merged**